tests/tutorials/tutorial_poisson_thermo.4C.yaml

```
@@ -0,0 +1,75 @@
+TITLE:
+  - "Tutorial on the solution of the Poisson equation using the thermal solver"
+PROBLEM SIZE:
+  ELEMENTS: 1024
+  NODES: 1089
+  MATERIALS: 1
+PROBLEM TYPE:
+  PROBLEMTYPE: "Thermo"
+THERMAL DYNAMIC:
+  DYNAMICTYPE: Statics
+  TIMESTEP: 1
+  NUMSTEP: 1
+  MAXTIME: 1
+  LINEAR_SOLVER: 1
+SOLVER 1:
+  SOLVER: "UMFPACK"
+  NAME: "direct linear solver"
+THERMAL DYNAMIC/RUNTIME VTK OUTPUT:
+  OUTPUT_THERMO: true
+  TEMPERATURE: true
+IO/RUNTIME VTK OUTPUT:
+  INTERVAL_STEPS: 1
+DESIGN LINE DIRICH CONDITIONS:
+  - E: 1
+    ENTITY_TYPE: node_set_id
+    NUMDOF: 1
+    ONOFF: [1]
+    VAL: [0]
+    FUNCT: [0]
+  - E: 2
+    ENTITY_TYPE: node_set_id
+    NUMDOF: 1
+    ONOFF: [1]
+    VAL: [0]
+    FUNCT: [0]
+  - E: 3
+    ENTITY_TYPE: node_set_id
+    NUMDOF: 1
+    ONOFF: [1]
+    VAL: [0]
+    FUNCT: [0]
+  - E: 4
+    ENTITY_TYPE: node_set_id
+    NUMDOF: 1
+    ONOFF: [1]
+    VAL: [0]
+    FUNCT: [0]
+DESIGN SURF NEUMANN CONDITIONS:
+  - E: 5
+    ENTITY_TYPE: node_set_id
+    NUMDOF: 1
+    ONOFF: [1]
+    VAL: [1]
+    FUNCT: [0]
+MATERIALS:
+  - MAT: 1
+    MAT_Fourier:
+      CAPA: 1
```

```suggestion
      CAPA: 1.0
```
same as above: As we use `1` to set ON/OFF in various places, I'd prefer here to have explicit `1.0` to show that a (float) number can/should be set here.